src/promptflow-devkit/tests/conftest.py

```diff
@@ -27,12 +30,30 @@
 from promptflow._utils.context_utils import _change_working_dir
 
 try:
+    from promptflow.recording.local import invoke_prompt_flow_service
     from promptflow.recording.record_mode import is_replay
 except ImportError:
     # Run test in empty mode if promptflow-recording is not installed
     def is_replay():
         return False
 
+    # copy lines from /src/promptflow-recording/promptflow/recording/local/test_utils.py
+    import time
+
+    from promptflow._cli._pf._service import _start_background_service_on_unix, _start_background_service_on_windows
```

Comment:
why duplicate?

Reply:
as offline synced, we need to ensure the invoke pfs function is available